watertap/unit_models/tests/test_reverse_osmosis_1D.py

```diff
@@ -9,1472 +9,817 @@
+
+import idaes.core.util.scaling as iscale
+
+from watertap.unit_models.reverse_osmosis_base import TransportModel
+
+import watertap.property_models.NaCl_prop_pack as props
+
+from watertap.unit_models.tests.unit_test_harness import UnitTestHarness
+
+from watertap.core import FrictionFactor
+
 
-import pytest
-from pyomo.environ import (
-    ConcreteModel,
-    value,
-    Constraint,
-    Var,
-    assert_optimal_termination,
-)
-from pyomo.util.check_units import assert_units_consistent
-
-from pyomo.network import Port
-from idaes.core import (
-    FlowsheetBlock,
-    MaterialBalanceType,
-    MomentumBalanceType,
-    StateBlock,
-)
 from watertap.unit_models.reverse_osmosis_1D import (
     ReverseOsmosis1D,
     ConcentrationPolarizationType,
     MassTransferCoefficient,
     PressureChangeType,
 )
-from watertap.unit_models.reverse_osmosis_base import TransportModel
-import watertap.property_models.NaCl_prop_pack as props
-
-from idaes.core.solvers import get_solver
-from idaes.core.util.model_statistics import (
-    number_variables,
-    number_unused_variables,
-    number_total_constraints,
-    degrees_of_freedom,
-)
 
-from idaes.core.util.testing import initialization_tester
-from idaes.core.util.scaling import (
-    calculate_scaling_factors,
-    unscaled_variables_generator,
-    badly_scaled_var_generator,
-)
-
-from watertap.core import (
-    MembraneChannel1DBlock,
-    FrictionFactor,
-)
-import idaes.logger as idaeslog
 
 # -----------------------------------------------------------------------------
 # Get default solver for testing
 solver = get_solver()
 # -----------------------------------------------------------------------------
 
 
-@pytest.mark.unit
-def test_config():
+def build():
     m = ConcreteModel()
     m.fs = FlowsheetBlock(dynamic=False)
+
     m.fs.properties = props.NaClParameterBlock()
-    m.fs.unit = ReverseOsmosis1D(property_package=m.fs.properties)
-
-    assert len(m.fs.unit.config) == 19
-    assert not m.fs.unit.config.dynamic
-    assert not m.fs.unit.config.has_holdup
-    assert m.fs.unit.config.material_balance_type == MaterialBalanceType.useDefault
-    assert m.fs.unit.config.momentum_balance_type == MomentumBalanceType.pressureTotal
-    assert not m.fs.unit.config.has_pressure_change
-    assert m.fs.unit.config.property_package is m.fs.properties
-    assert m.fs.unit.config.pressure_change_type is PressureChangeType.fixed_per_stage
-    assert (
-        m.fs.unit.config.concentration_polarization_type
-        is ConcentrationPolarizationType.calculated
+
+    m.fs.unit = ReverseOsmosis1D(
+        property_package=m.fs.properties,
+        has_pressure_change=True,
+        concentration_polarization_type=ConcentrationPolarizationType.calculated,
+        mass_transfer_coefficient=MassTransferCoefficient.calculated,
+        pressure_change_type=PressureChangeType.calculated,
+        transformation_scheme="BACKWARD",
+        transformation_method="dae.finite_difference",
```

Comment:
```suggestion
```
If we remove these lines for one of the `build` functions, then this PR will have no change in coverage. It won't affect how the model is built since these are the defaults.
https://app.codecov.io/gh/watertap-org/watertap/pull/1301/indirect-changes